example_configs/nextcloud_oidc_authelia.md

```diff
@@ -0,0 +1,87 @@
+## Assumptions
+
+If you're here, there are some assumptions being made about access and capabilities you have on your system:
+1. You have Authelia up and running, understand its functionality, and have read through the documentation. 
+2. You have [LLDAP](https://github.com/nitnelave/lldap) up and running. 
+3. You have Nextcloud and LLDAP communicating and without any config errors. See the [example config for Nextcloud](example_configs/nextcloud.md)
```

Comment:
The link for the Nextcloud config is relative, so it should be just `nextcloud.md`, not `example_configs/nextcloud.md`